spec.bs

```diff
@@ -5804,18 +5899,35 @@ enum KAnonStatus { "passedAndEnforced", "passedNotEnforced", "belowThreshold", "
   <dt>{{ReportWinBrowserSignals/madeHighestScoringOtherBid}}
   <dd>True if the interest group owner was the only bidder that made bids with the second highest
     score
+  <dt>{{ReportWinBrowserSignals/selectedBuyerAndSellerReportingId}}

```

Comment:
Similarly, no need for either `ReportWinBrowserSignals/selectedBuyerAndSellerReportingId` nor `ReportWinBrowserSignals/buyerAndSellerReportingId`.